homeassistant/components/unifiprotect/__init__.py

```diff
@@ -41,6 +44,60 @@
 SCAN_INTERVAL = timedelta(seconds=DEFAULT_SCAN_INTERVAL)
 
 
+async def _async_migrate_data(
+    hass: HomeAssistant, entry: ConfigEntry, protect: ProtectApiClient
+) -> None:
+
+    registry = er.async_get(hass)
+    to_migrate = []
+    for entity in er.async_entries_for_config_entry(registry, entry.entry_id):
```

Comment:
We try to differentiate between `Entity` instances and entity registry `RegistryEntry` instances, by only naming `Entity` instances `entity` and registry entries `entity_entry` or similar.